repo-agent/review-sandbox/main.go

```diff
@@ -63,15 +84,27 @@ func runReview() error {
 		return fmt.Errorf("unknown agent name: %s", agentName)
 	}
 
-	var output []byte
-	var err error
-	maxRetries := 3
-	for i := 0; i < maxRetries; i++ {
-		log.Printf("Running Agent %s (attempt %d/%d)", agentName, i+1, maxRetries)
-		output, err = agentFn()
+	var accumulatedAgentOutput AgentOutput
+	maxRuns := 10
```

Comment:
For each run would it be worth outputting the comments left by the previous run as part of the context to the next call to increase the odds new comments are left?